kani-driver/src/args.rs

```diff
@@ -273,6 +266,51 @@ impl KaniArgs {
     }
 }
 
+/// Arguments that Kani pass down into Cargo essentially uninterpreted.
+/// These generally have to do with selection of packages or activation of features.
+/// These do not (currently) include cargo args that kani pays special attention to:
```

Comment:
Should it? I wasn't sure. I didn't want to make too strong of a statement if we decide otherwise later except someone sees this comment and wonders "well, why not?"...

Reply:
I think you misunderstood the comment? It's not about `kani` vs `cargo-kani`, it's about cargo args that kani is interpreting (hence the examples like `--target-dir`) versus those that we just kinda pass down.

Reply:
ok, I literally just said "kani is interpreting" again, which was maybe the confusion (between kani-the-project, and kani-the-binary)

"It's about cargo args that we are interpreting... versus those that we just pass down"

Reply:
Correct, but this PR doesn't change that yet. There's more extensive work needed (e.g. in how arguments get plumbed through the session) to do that.